src/main/car/functionality/DealershipUI

```diff
@@ -0,0 +1,131 @@
+import javax.swing.*;
+
+public class DealershipUI {
+	static String dealerID = "";
+	
+	private static void makeDealerGUI() {
```

Comment:
It looks like this is what creates the frame, so in the parameter, accept a DealerGroup. This will help you find the specific dealer you need.

Reply:
I'll give it a try and build the button functionality around that parameter. That suggestion addressed the main question I had about this class

Reply:
Note to self: Consider removing the dealerID string attribute and feed the output of the feedText method into the search method of DealerGroup

Reply:
Added parameter to the makeDealerGUI method. Replaced String field with a Dealership field that starts as null.